Front-end/Auction_App_Front/app/components/save-user-card.js

```diff
@@ -0,0 +1,85 @@
+import Component from '@ember/component';
+import {inject as service} from '@ember/service';
+
+export default Component.extend({
+  session: service('session'),
+  stripeHttp: service(),
+  notifications: service('notification-messages'),
+  userEmail: null,
+  cardName: '',
+  init() {
+    this._super(...arguments);
+    this.set('userEmail', this.get('session.data.email'));
+  },
+  didInsertElement() {
+    const stripe = Stripe('key');
+    const elements = stripe.elements();
+    const errorElement = document.querySelector('.error');
+    const self = this;
+    const style = {
+      base: {
+        fontFamily: 'Lato-Regular',
+        fontSize: '16px',
+        color: '#252525',
+        letterSpacing: '0.56px',
+        '::placeholder': {
+          fontFamily: 'Lato-Regular',
+          fontSize: '18px',
+          color: '#9B9B9B',
+          letterSpacing: '0.63px',
+        },
+      },
+    };
+    const cardNumberElement = elements.create('cardNumber', {
```

Comment:
Try using ember-stripe-elements. Also, a lot of duplicate code from payment-form component.